src/cclib/io/cmlwriter.py

```diff
@@ -43,6 +43,9 @@ def generate_repr(self):
         if self.jobfilename is not None:
             d['id'] = self.jobfilename
         _set_attrs(molecule, d)
+        
+        if hasattr(self.ccdata, 'atomnos'):
+            elements = [self.pt.element[Z] for Z in self.ccdata.atomnos]
```

Comment:
@TDI-tenderholt yes, thats the only place it was used. I made the change and squashed the commit.